Kickstarter-iOS/Views/Controllers/SettingsNotificationsViewController.swift

```diff
@@ -89,13 +90,37 @@ internal final class SettingsNotificationsViewController: UIViewController {
   }
 
   private func animatePickerView(isHidden: Bool) {
-    UIView.animate(withDuration: 0.25) {
-      self.emailPickerViewTopConstraint.constant = isHidden
-        ? 0 : -SettingsNotificationsViewController.emailPickerViewHeight
-
-      self.view.setNeedsLayout()
-      self.view.layoutIfNeeded()
-    }
+    UIView.animate(
+      withDuration: 0.25,
+      animations: { [weak self] in
+        guard let self = self else { return }
+
+        if !isHidden && AppEnvironment.current.isVoiceOverRunning() {
+          // Tells VoiceOver to ignore other elements in the same parent view
+          self.emailFrequencyPickerView.accessibilityViewIsModal = true
+
+          UIAccessibility.post(
+            notification: UIAccessibility.Notification.screenChanged,
+            argument: self.emailFrequencyPickerView
+          )
+        }
+
+        self.emailPickerViewTopConstraint.constant = isHidden ? 0 : self.emailFrequencyPickerView.frame.height
+        self.view.setNeedsLayout()
```

Comment:
Have you tested without `setNeedsLayout()`? I think we only need `layoutIfNeeded()`.

Reply:
Not originally my code...but have tried removing and it looks like everything works. Agree that we should only need to call `layoutIfNeeded()`.